include/infinispan/hotrod/Configuration.h

```diff
@@ -54,6 +54,27 @@ class Configuration
                 socketTimeout(_socketTimeout), sslConfiguration(_sslConfiguration),tcpNoDelay(_tcpNoDelay),
                 valueSizeEstimate(_valueSizeEstimate), maxRetries(_maxRetries), balancingStrategyProducer(bsp) {}
 
+    Configuration(const std::string &_protocolVersion,
```

Comment:
I think the other constructor can be removed, since the builder will use this one now ?